features/step_definitions/registry.rb

```diff
@@ -371,3 +371,22 @@
   cb_name ||= :registry_route
   cb[cb_name] = api_hostname.gsub("api","registry")
 end
+
+# Generate default route
+Given /^I enable image-registry default route$/ do
+  ensure_admin_tagged
```

Comment:
is not this a configuration change that can easily interfere with other tests? In such case it needs to be `ensure_destructive_tagged` even if it will not break the cluster. Also registering clean-up to revert to original state must be part of the step.

Reply:
This step doesn't make any image-registry pod restart and won't influence ongoing tests. If the route is enabled at the first place, it doesn't matter this command executed again, usually, user needs this route enabled.